packages/fluxible-addons-react/FluxibleComponent.js

```diff
@@ -2,6 +2,7 @@
  * Copyright 2015, Yahoo Inc.
  * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
  */
+/*eslint no-unused-vars:0 */
```

Comment:
Why did this fail?

Reply:
Because of:
```
function FluxibleComponent(props, context) {
    React.Component.apply(this, arguments);
}
```

`props` and `context` are not used. I wanted to leave those in to help document the API. So I just disabled the rule for that file.